scanner/config.yaml.sample

```diff
@@ -6,11 +6,17 @@ indexer:
     conn_string: "host=/var/run/postgresql"
     password_file: ""
   get_layer_timeout: 1m
+  repo:
+    rhel-repository-scanner:
+        repo2cpe_mapping_url: https://sensor.stackrox.svc/scanner/definitions?type=repo2cpe
```

Comment:
If adding `updater.enabled`, I think the sample should use `updater.enabled=false`, and leave the URL empty (`""`). I think the README uses the .sample as a starting point for local development, that's why.